packages/codemods/nextAuth/useSession-array-to-Object-destrucutring/.codemodrc.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://codemod-utils.s3.us-west-1.amazonaws.com/configuration_schema.json",
+  "name": "nextAuth/v4/useSession-array-to-object-destructuring",
```

Comment:
Lets change the name to `nextAuth/4/useSession-array-to-object-destructuring`

Reply:
Sure✌️